package.json

```diff
@@ -53,6 +53,8 @@
     "@storybook/manager-webpack5": "^6.4.9",
     "@storybook/preset-scss": "^1.0.3",
     "@storybook/react": "^6.4.9",
+    "@testing-library/dom": "^9.2.0",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
Where are these used?

Reply:
I thought about firing user events in one of the tests I have written, but I refrained from doing so. Hence these dependencies will no more be useful and I will remove these in upcoming commit 👍🏻